source/distributed/repartitioning_policy_tools.cc

```diff
@@ -206,10 +206,10 @@ namespace RepartitioningPolicyTools
 
     // step 1) check if all processes have enough cells
 
-    unsigned int n_locally_owned_active_cells = 0;
-    for (const auto &cell : tria_in.active_cell_iterators())
-      if (cell->is_locally_owned())
-        ++n_locally_owned_active_cells;
+    const auto locally_owned_cells =
+      tria_in.active_cell_iterators() | IteratorFilters::LocallyOwnedCell();
+    const unsigned int n_locally_owned_active_cells =
+      std::distance(locally_owned_cells.begin(), locally_owned_cells.end());
```

Comment:
If you want, you can use `std::count_if` here, too.

Reply:
Yes, so done!